client/src/components/layout/ReviewTile.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const ReviewTile = ({ userName, rating, description }) => {
+  return (
+    <div className="grid-x callout">
+      <div className="cell small-9">
+        <h3>{userName}</h3>
+        <p>{description}</p>
+      </div>
+      <div className="cell small-3">
+        <div className="dog-rating">{rating} / 10</div>
+      </div>
```

Comment:
We have lot's of div's wrapped around div's of div's here lol let's try to cut down the number of div nested when designing our layout so that we don't clutter the DOM with elements

Reply:
We'll need all of them to complete the design, but point taken. We should omit until they're actually needed.

Reply:
removed